src/js/modules/infragistics.ui.combo.js

```diff
@@ -1620,6 +1621,9 @@
             };
 
             this._analyzeOptions();
+
+			// Options related to localization
+			this._changeLocale();
```

Comment:
Looks like a small indentation issue. Could be because the other lines use spaces.